packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/resiliation/resiliation.routing.js

```diff
@@ -1,4 +1,10 @@
 import { BillingService as Service } from '@ovh-ux/manager-models';
+import set from 'lodash/set';
+
+const END_STRATEGY = {
+  CANCEL_SERVICE: 'CANCEL_SERVICE',
+  STOP_ENGAGEMENT: 'STOP_ENGAGEMENT_FALLBACK_DEFAULT_PRICE',
+};
```

Comment:
Maybe this could be retrieved by API, through this enum services.billing.engagement.EndStrategyEnum, to avoid maintaining these constants?